src/ISADotnet/InvestigationFile/API.fs

```diff
@@ -1,640 +1,634 @@
 ﻿namespace ISADotNet.InvestigationFile
```

Comment:
The functions need to be resorted. It does not get clear from the file and module structur which function is related to studies and which is related to investigations

Reply:
This file is only left in as depricated  to get the previously done changes into the commit history. I did this to have a failback safe in case i missed to redistribute something or did an error during the merge of your newest commits. So if something went wrong we have this file with some of the updates still in the commit history. This is also the reason why i added the obsolete tag to the API module. I hope this gives some additional insight. Let me know what you think.